docs/data/material/guides/understand-mui-packages/understand-mui-packages.md

```diff
@@ -37,7 +37,7 @@ The following chart illustrates how MUI's packages are related to one another:
 <img src="/static/images/packages/mui-packages.png" style="width: 796px; margin-top: 4px; margin-bottom: 8px;" alt="The first half of the image shows @mui/material and @mui/base as component libraries, and @mui/system and styled engines as styling solutions, both under the MUI Core umbrella. The second half shows @mui/x-data-grid and @mui/x-date-pickers as components from MUI X."/>
 
 In this article, we'll only cover the MUI Core packages.
-Check the [MUI X overview](/x/getting-started/overview) for more information about our collection of advanced components.
+Check the [MUI X overview](/x/advanced-components/) for more information about our collection of advanced components.
```

Comment:
```suggestion
Check the [MUI X overview](/x/overview/) for more information about our collection of advanced components.
```

Should we change the URL for this page? 🤔

Reply:
For now, it's a 404 https://mui.com/x/overview/ but agree, I believe @samuelsycamore work on this in https://github.com/mui/mui-x/pull/4692, to consider the change if merged.